hack/ci/cloud-init/controller.yaml.tpl

```diff
@@ -70,7 +70,7 @@
     IMAGE_URLS+="https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/cirros/2022-12-05/cirros-0.6.1-x86_64-disk.img,"
     IMAGE_URLS+="https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/ubuntu/2024-05-28/ubuntu-2204-kube-v1.29.5.img,"
     IMAGE_URLS+="https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/ubuntu/2024-05-28/ubuntu-2204-kube-v1.30.1.img,"
-    IMAGE_URLS+="https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/flatcar/flatcar-stable-3815.2.2-kube-v1.30.1.img,"
+    IMAGE_URLS+="https://wfwfg.s3.pl-waw.scw.cloud/flatcar-stable-3975.2.0-kube-v1.30.1.img,"
```

Comment:
please fix the URL here to use google servers, I'm taking care of uploading the image.

Reply:
pushed, URL is:
https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/flatcar/flatcar-stable-3975.2.0-kube-v1.30.1.img